protocols/rendezvous/src/client.rs

```diff
@@ -104,7 +107,10 @@ impl Behaviour {
     /// Unregister ourselves from the given namespace with the given rendezvous peer.
     pub fn unregister(&mut self, namespace: Namespace, rendezvous_node: PeerId) {
         self.inner
-            .send_request(&rendezvous_node, Unregister(namespace));
+            .send_request(&rendezvous_node, Unregister(namespace.clone()));
```

Comment:
Not that it matters much but can we avoid the `clone` if we remove it from the other hashmap first?